lib/Backend/JITTimePolymorphicInlineCacheInfo.cpp

```diff
@@ -53,8 +53,9 @@ JITTimePolymorphicInlineCacheInfo::InitializePolymorphicInlineCacheInfo(
     __in Js::PolymorphicInlineCacheInfo * runtimeInfo,
     __out PolymorphicInlineCacheInfoIDL * jitInfo)
 {
+#pragma warning(suppress: 6001)
```

Comment:
Seems strange. "WriteBarrierPtr" has default constructor which initialize ptr to null, thus inside "operator=" there is no uninitialized memory.